python_scripts/01_tabular_data_exploration.py

```diff
@@ -259,8 +271,8 @@ def plot_tree_decision_function(tree, X, y, ax=None):
     data_subset[target_column].to_numpy())
 
 # %% [markdown]
-# We will create a decision tree which we will keep really simple on purpose.
-# We will only allow a maximum of 2 rules to interpret the results.
+# We will create a simple decision tree with a maximum of 2 rules, in order
```

Comment:
I am thinking that we could avoid using the term `decision tree` and instead referring as partitioning or splitting the feature space?

Reply:
I am going to leave this one and trust the wording will be changed when we include a picture rather than create the DecisionTree in the notebook explicitly